<launch>
    <arg name="robot_name" default="sam" />
    <node pkg="driver_stim300" exec="stim300_driver" name="stim300_driver" output="screen" namespace = "$(var robot_name)">
        <param name = "device_name" value="/dev/ttyUSB0" />
        <param name =  "stanard_deviation_of_gyro" value="0.0004" />
        <param name = "stanard_deviation_of_acc" value="0.004" />
        <param name = "sample_rate" value = "125" />

    </node>
</launch>
